src/components/toasts/index.tsx

```diff
@@ -0,0 +1,179 @@
+import {
```

Comment:
I'll approve because we need it asap but we should have this over the library right?